terraform/digital.gov.tf

```diff
@@ -820,40 +804,29 @@ resource "aws_route53_record" "app_touchpoints_digital_gov_cname" {
   }
 }
 
-# Add SPF record for mail subdomain
-resource "aws_route53_record" "app_touchpoints_digital_gov_spf" {
-  zone_id = aws_route53_zone.digital_toplevel.zone_id
-  name    = "_acme-challenge.app.touchpoints.digital.gov."
-  type    = "CNAME"
-  ttl     = 120
-  records = ["_acme-challenge.app.touchpoints.digital.gov.external-domains-production.cloud.gov."]
-}
-
-resource "aws_route53_record" "app_touchpoints_digital_gov_ses_cname_1" {
+# Mail handling for app subdomain - using a dedicated mail subdomain
+resource "aws_route53_record" "mail_app_touchpoints_digital_gov_mx" {
   zone_id         = aws_route53_zone.digital_toplevel.zone_id
-  name            = "qqtoqzlc5a24irzufsu4lbdpoc3mvr3n._domainkey.app.touchpoints.digital.gov"
-  type            = "CNAME"
-  ttl             = 1800
-  allow_overwrite = true  # Add this to handle conflicts
-  records         = ["qqtoqzlc5a24irzufsu4lbdpoc3mvr3n.dkim.amazonses.com"]
-}
+  name            = "mail-app.touchpoints.digital.gov."  # Dedicated mail subdomain
```

Comment:
the `mail.touchpoints.digital.gov` seems preferable to to this one.  

i'm thinking, either:

1. one `mail.` subdomain, with different emails for each environment (staging, demo, prod), or
1. one `mail-ENV.` subdomain for each environment.

this `mail-app` could work for prod, but isn't ideal.

Reply:
In the app config you can use `user+prod@mail.`  to denote  ENV on the same mail server address